analyses/molecular-subtyping-EPN/03-summary.Rmd

```diff
@@ -133,13 +133,13 @@ viewDataTable(EPN_final %>%
 
 ```{r}
 EPN_final <- EPN_final %>%
-  mutate(subgroup = ifelse(subgroup == "EPN, To be classified" & `1q_gain` > 0 & 
-                             (CXorf67_expr_zscore > 3 | TKTL1_expr_zscore > 3), 
-                           yes = "EPN, PF A", 
-                           no = ifelse(subgroup == "EPN, To be classified" & 
-                                         (`6q_loss` > 0 | `6p_loss` > 0) & 
-                                         (GPBP1_expr_zscore > 3 | IFT46_expr_zscore > 3), yes = "EPN, PF B", no = subgroup))) 
-
+  mutate(subgroup = case_when(
+    subgroup == "EPN, To be classified" & `1q_gain` > 0 & (CXorf67_expr_zscore > 3 | TKTL1_expr_zscore > 3) ~  "EPN, PF A",
```

Comment:
```suggestion
    subgroup == "EPN, To be classified" & `1q_gain` > 0 & TKTL1_expr_zscore > 3 ~  "EPN, PF A",
```
probably can do this.. I thought we had a sample which should come back to be PF A, but maybe not

Reply:
@jharenza - committed the suggested logic. Ready for another look!